src/software/ai/hl/stp/tactic/dribble/dribble_fsm.h

```diff
@@ -0,0 +1,220 @@
+#pragma once
+
+#include "shared/constants.h"
+#include "software/ai/evaluation/pass.h"
+#include "software/ai/hl/stp/tactic/move/move_fsm.h"
+#include "software/ai/hl/stp/tactic/tactic.h"
+#include "software/ai/intent/move_intent.h"
+#include "software/geom/algorithms/acute_angle.h"
+#include "software/geom/algorithms/contains.h"
+#include "software/geom/algorithms/distance.h"
+
+struct DribbleFSM
+{
+    class GetPossessionState;
```

Comment:
Curious why we need to get rid of the `GetPossessionFSM`? Can `DribbleFSM` not be a HFSM with `GetPossessionFSM` being the SubFSM?

Reply:
With the dribble fsm, I couldn't think of use cases for the GetPossessionFSM. Making an HFSM would've resulted in a lot of extra boilerplate compared to just combining them